mmtrack/datasets/base_sot_dataset.py

```diff
@@ -177,13 +182,19 @@ def prepare_test_data(self, video_ind, frame_ind):
         Returns:
             dict: testing data of one frame.
         """
-        ann_infos = self.get_ann_infos_from_video(video_ind)
-        img_infos = self.get_img_infos_from_video(video_ind)
+        if self.test_memo.get('video_ind', -1) != video_ind:
```

Comment:
`if self.test_memo.get('video_ind', None) is not None:`

The video_ind can be equal to -1